modules/ROOT/pages/policy-mule4-detokenization.adoc

```diff
@@ -132,22 +142,17 @@ Only expressions using dots as field separators are allowed. While `#[payload.pe
 | No. +
 Keep in mind that if you don't specify a selector expression, the policy will be applied, but won't detokenize anything from your payload.
 
-| *Format*
+| Format
 | Type of information that will be detokenized. Formats may vary depending on the selected service. Choose one format from the list.
 
 If you do not see any formats in the list, check that the tokenization service selected contains at least one format. +
 See the xref:anypoint-security::tokenization-formats.adoc[Tokenization Formats] article for more information.
 | No. +
-Keep in mind that if you don't specify a format, the policy will be applied, but won't detokenize anything from your payload.
+If you do not specify a format, the policy will be applied. However, nothing is detokenized from your payload.
 
 |===
 
-== Scope and Considerations
 
-* One single instance of the policy supports only one tokenization service. All the formats that you configured need to be contained in the same tokenization service.
-* One single instance of the policy can be used to detokenize a value either in the request or the response, but not both.
-* The policy is not aware of the state of the tokenization service until runtime: If by mistake, you delete your tokenization service, or if the service is down for some reason, the policy won't be able to establish a communication with the service, causing all the requests to fail until the service is available again.
-* During policy configuration, your *services* drop-down menu only displays the tokenization services that are ready to use. If you do not see the service you want, check if it is already deployed.
 
 == See Also
 
```

Comment:
Same comment as with the client id enforcement, we should add the link to the release notes of this policy here.

Reply:
Added a link! Thanks!